src/QirRuntime/test/QIR-static/qir-test-math.qs

```diff
@@ -60,5 +60,9 @@ namespace Microsoft.Quantum.Testing.QIR.Math {
         return 0;
     }
 
+    operation DrawRandomIntTest(min : Int, max : Int) : Int {
+        return DrawRandomInt(min, max);
+    }
```

Comment:
Thanks for catching `DrawRandomIntTest` name mismatch. Fixed.

:-) The motivation is that _I'm a newbie in Q#_ and 
(a) I'm not sure if logic of that complexity is implementable with the current means available in Q#, and 
(b) it will take me a few times more time to write that in Q# rather than in C++.
Having said that, and keeping in mind that @irinayat-MS is waiting for my CR of her PR (and I have a number of other items on my plate) I decided to do that in C++. Let me know if that needs to be re-written in Q#.